src/stuf/stuf_zds/client.py

```diff
@@ -218,7 +219,7 @@ def create_zaak_identificatie(self) -> str:
     def create_zaak(
         self,
         zaak_identificatie: str,
-        zaak_data: dict,
+        zaak_data: MutableMapping[str, Any],
```

Comment:
```suggestion
        zaak_data: MutableMapping[str, object],
```
recently Chris pointed out that `object` is a better alternative than `Any` - it will still type check because everything is an object in python, but it doesn't taint the types in the rest of the codebase.

@CharString any chance you can provide an example where the `Any` messes up things?

Reply:
I used `Any` because it's more flexible (for example when I do `for partner in zaak_data[RegistrationAttribute.partners]` the type checker complains that `object` is not iterable).

Reply:
okay for now then, hopefully we'll be able to clean that up later :)